### PR TITLE
fix: mobile support for multi-round UI

### PR DESCRIPTION
**Pull request by kittybest**
**Description**
related to PR #399 
This is the rebase main branch version.

### CODE REVIEW

File: packages/interface/src/components/Footer.tsx [hunk 1]
```diff
@@ -31,13 +31,13 @@ export const Footer = (): JSX.Element => (
 
     <div className="flex justify-end gap-4">
       <a className="flex items-center gap-1" href="https://maci.pse.dev" rel="noreferrer" target="_blank">
-        <span>Documentation</span>
+        <span>Doc</span>
```
Comment (ctrlc03) [inline packages/interface/src/components/Footer.tsx:34]: Docs?

**Comment by ctrlc03** [comment]
think this is a feat not a fix @kittybest @crisgarner?

**Comment by kittybest** [comment]
> think this is a feat not a fix @kittybest @crisgarner?

Both works for me, how do you think?

**Comment by kittybest** [comment]
> > think this is a feat not a fix @kittybest @crisgarner?
> 
> Both work for me, how do you think?

**Review by crisgarner — APPROVED** [review]

**Pull request merged**